public/js/editcube.js

```diff
@@ -514,6 +506,131 @@ $('#showTagColorsCheckbox').change(function(e) {
   });
 });
 
+$('#applyAdvancedFilterButton').click(function(e) {
+  console.log('click');
+  e.preventDefault();
+
+  var str = '';
+
+  if($('#filterName').val().length > 0)
+  {
+    str += 'n:'+$('#filterName').val();
+  }
+
+  if($('#filterOracle').val().length > 0)
+  {
+    var split = $('#filterOracle').val().split(' ');
+    split.forEach(function(val, index)
+    {
+      str += ' o:'+val;
+    });
+  }
+
+  if($('#filterCMC').val().length > 0)
+  {
+    if($('#filterCMCOp').val() == '!')
+    {
+      str += ' -cmc='+$('#filterCMC').val();
+    }
+    else
+    {
+     str += ' cmc'+$('#filterCMCOp').val()+$('#filterCMC').val();
+    }
+  }
+  if($('#filterPower').val().length > 0)
+  {
+    if($('#filterPowerOp').val() == '!')
+    {
+      str += ' -pow='+$('#filterPower').val();
+    }
+    else
+    {
+     str += ' pow'+$('#filterPowerOp').val()+$('#filterPower').val();
+    }
+  }
+  if($('#filterToughness').val().length > 0)
+  {
+    if($('#filterToughnessOp').val() == '!')
+    {
+      str += ' -tou='+$('#filterToughness').val();
+    }
+    else
+    {
+     str += ' tou'+$('#filterToughnessOp').val()+$('#filterToughness').val();
+    }
+  }
+
+  //Color
+  var colorStr = '';
+  ['W','U','B','R','G','C'].forEach(function(val, index) {
+    if($('#filterColor' + val).prop('checked'))
+    {
+      colorStr += val;
+    }
+  });
+  if(colorStr.length > 0)
+  {
+    str += ' c' + $('#filterColorOp').val() + colorStr;
+  }
+  //Color Identity
+  colorStr = '';
+  ['W','U','B','R','G','C'].forEach(function(val, index) {
+    if($('#filterColorIdentity' + val).prop('checked'))
+    {
+      colorStr += val;
+    }
+  });
+  if(colorStr.length > 0)
+  {
+    str += ' ci' + $('#filterColorIdentityOp').val() + colorStr;
+  }
+  //Mana
+  if($('#filterMana').val().length > 0)
+  {
+    str += ' m:'+$('#filterMana').val();
+  }
+
+  //Type
+  if($('#filterType').val().length > 0)
+  {
+    str += ' t:'+$('#filterType').val();
+  }
+
+  //tags
+  if($('#filterTag').val().length > 0)
+  {
+    str += ' tag:"'+$('#filterTag').val()+'"';
+  }
+
+  //price
+  if($('#filterPrice').val().length > 0)
+  {
+    str += ' p'+$('#filterPriceOp').val()+$('#filterPrice').val();
+  }
+  
+  //price foil 
+  if($('#filterPriceFoil').val().length > 0)
+  {
+    str += ' pf'+$('#filterPriceFoilOp').val()+$('#filterPriceFoil').val();
+  }
+
+  //status
+  if($('#filterStatus').val().length > 0)
+  {
+    str += ' stat:"'+$('#filterStatus').val()+'"';
+  }
+
+  //loyalty
```

Comment:
If these comments are meant to indicate TODO items, they would be better as one or more GitHub issues. Then, they could be removed from this code.

Reply:
More as comments to myself before I got onto my plane, I'll create an issue for these fields